docs/compilers/Deterministic Inputs.md

```diff
@@ -17,15 +16,17 @@ The following are considered inputs to the compiler for the purpose of determini
   - the strong name key file
   - `@` response files
   - Analyzers
+  - Generators
   - Rulesets
-  - "additional files" that may be used by analyzers
-- The current culture (for the language in which diagnostics and exception messages are produced).
+  - "additional files" that may be used by analyzers and generators
+- The current culture if `/preferuilang` is not specified (for the language in which diagnostics and exception messages are produced).
 - The current OS code page if `/codepage` is not specified and any of the input source files do not have BOM and are not UTF-8 encoded.
 - The existence, non-existence, and contents of files on the compiler's search paths (specified, e.g. by `/lib` or `/recurse`)
 - The CLR platform on which the compiler is run:
   - The result of `double` arithmetic performed for constant-folding may use excess precision on some platforms.
   - The compiler uses Unicode tables provided by the platform.
 - The version of the zlib library that the CLR uses to implement compression (when `/embed` or `/debug:embedded` is used).
-- The value of `%LIBPATH%`, as it can affect analyzer dependency loading.
+- The value of `%LIBPATH%`, as it can affect reference discovery if not fully qualified and analyzer / generator dependency loading.
```

Comment:
> if not fully qualified [](start = 64, length = 23)

This clause makes it difficult to parse the subsequent "and analyzer / generator dependency loading". Perhaps add parentheses: "(if not fully qualified)".

Reply:
I clarified the language here. The generator / analyzer dependency isn't about whether or not they're fully qualified but rather how the runtime deals with analyzer / generator dependency loading.